### PR TITLE
AmmoHUD Updates

### PR DESCRIPTION
**Pull request by Tupinambis**
## About The Pull Request

New and Improved AmmoHUD sprites.
Removal of unused sprites.
New sprites for certain ammo types.

## Why It's Good For The Game

Visual clarity is nice, and I like pretty sprites.

## Changelog

:cl:
imageadd: Updated and new ammoHUD sprites.
imagedel: Unused ammoHUD sprites
/:cl:

### CODE REVIEW

**Comment by Blundir** [comment]
dreammaker_A4JHUgiyN3

**Comment by Lumipharon** [comment]
Mad man flipping all the ammo icons.
You missed a few though, like shotgun shells.

Also at least some of the ones you're deleting are in use, like shell_apcr

**Comment by Tupinambis** [comment]
> dreammaker_A4JHUgiyN3

fixed

**Comment by Tupinambis** [comment]
> Mad man flipping all the ammo icons. You missed a few though, like shotgun shells.
> 
> Also at least some of the ones you're deleting are in use, like shell_apcr

Shotgun shells are already oriented to the right 

> Mad man flipping all the ammo icons. You missed a few though, like shotgun shells.
> 
> Also at least some of the ones you're deleting are in use, like shell_apcr

Also fixed, probably. Didn't see shell_apcr since it was in a different spot.

**Review by TiviPlus — APPROVED** [review]

File: code/modules/projectiles/ammo_datums.dm [hunk 1]
```diff
@@ -705,6 +706,24 @@ GLOBAL_LIST_INIT(no_sticky_resin, typecacheof(list(/obj/item/clothing/mask/faceh
...
+
+/datum/ammo/bullet/rifle/standard_dmr/incendiary
+	name = "incendiary marksman bullet"
+	hud_state = "hivelo_fire"
+	hud_state_empty = "hivelo_empty"
+	flags_ammo_behavior = AMMO_BALLISTIC|AMMO_INCENDIARY
+	damage = 35
+	sundering = 0 // incen doens't have sundering
+	accuracy = -10
+
```
Comment (Lumipharon) [inline code/modules/projectiles/ammo_datums.dm:726]: these ammo types are unused and were removed from the code.

**Pull request closed without merging**